src/sentry/models/sentryapp.py

```diff
@@ -33,6 +34,11 @@ class SentryApp(ParanoidModel, HasApiScopes):
 
     name = models.TextField()
     slug = models.CharField(max_length=64, unique=True)
+    status = BoundedPositiveIntegerField(
+        default=SentryAppStatus.UNPUBLISHED,
+        choices=SentryAppStatus.as_choices(),
+        db_index=True,
+    )
```

Comment:
FYI @mattrobenolt @JTCunning – Just a heads up since this is going to run a migration. There's no records in this table currently, though, so shouldn't affect anything.

Reply:
If you’re 100% sure, then this is fine.